experimental/LieAlgebras/src/LieAlgebraModuleHom.jl

```diff
@@ -403,7 +403,7 @@ function canonical_projection(V::LieAlgebraModule, i::Int)
   @req fl "Module must be a direct sum"
   @req 1 <= i <= length(Vs) "Index out of bound"
   j = sum(dim(Vs[l]) for l in 1:(i - 1); init=0)
-  proj = hom(
+  projection = hom(
```

Comment:
What is the purpose of this change and the similar ones in parts of the code that have nothing to do with schemes? I don't really see why they are necessary.

Reply:
https://github.com/oscar-system/Oscar.jl/issues/3345#issuecomment-1950227788